.config/webpack/base.js

```diff
@@ -4,10 +4,17 @@ const packagejson = require('./../../package.json');
 
 const dashLibraryName = packagejson.name.replace(/-/g, '_');
 
-module.exports = (preprocessor = {}, mode = 'development') => {
+module.exports = (options = {}) => {
+    const babel = options.babel || undefined;
+    const preprocessor = options.preprocessor || {};
+    const mode = options.mode || 'development';
+    const ts = options.ts || {};
+
     console.log('********** Webpack Environment Overrides **********');
     console.log('Preprocessor', JSON.stringify(preprocessor));
     console.log('mode', mode);
+    console.log('babel', JSON.stringify(babel));
+    console.log('ts', JSON.stringify(ts));
```

Comment:
Have to pass in ts and babel configurations explicitly otherwise they get resolved through some mechanism that does not behave the way I need it to!